draft-ietf-tls-esni.md

```diff
@@ -116,6 +116,68 @@ document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here. All TLS
 notation comes from {{RFC8446, Section 3}}.
 
+# Abbreviations
+
+This section expands abbreviations that are not 'well known' as per RFC Editors abbreviations list.
+
+AAD
+: Additional Authenticated Data
+: Associated Authenticated Data
+
+AEAD
+: Authenticated Encryption with Associated Data
+
+ALPN
+: Application-Layer Protocol Negotiation
+
+CA
+: Certificate Authority
+
+CH
```

Comment:
`CH` + `HRR` + `EE`, as mentioned in the "Update of the TLS ExtensionType Registry" section, are TLS handshake messages defined in RFC8446.
CH = Client Hello
HRR = Hello Retry Request
EE = Encrypted Extensions

Reply:
And here it is, that I made a mistake on HRR because it collides too with HTTP RR so this is a perfect case that for a non TLS implementor it is hard to follow and just adds extra confusion.